1036-timeline-design-guidance.md

```diff
@@ -0,0 +1,47 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-17
+updated: 2024-10-21
+---
+
+# Timeline Component
+
+The [Timeline](https://clarity.design/documentation/timeline) component is a visual indicator that shows progress through a workflow. It can also guide the user through specific processes or pre-defined steps.
```

Comment:
```suggestion
The [timeline](https://clarity.design/documentation/timeline) component is a visual indicator that shows progress through a workflow. It can also guide the user through specific processes or pre-defined steps.
```